packages/api/package.json

```diff
@@ -37,17 +37,17 @@
     "@nestjs/schematics": "^8.0.0",
     "@nestjs/testing": "^8.0.0",
     "@types/express": "^4.17.13",
```

Comment:
Nit, but maybe we should take the time to lockdown all these packages

Reply:
Probably smart, thanks!